response_operations_ui/__init__.py

```diff
@@ -21,10 +23,21 @@
 app.config.from_object(app_config)
 
 app.url_map.strict_slashes = False
+app.secret_key = app.config['RESPONSE_OPERATIONS_UI_SECRET']
 
 logger_initial_config(service_name='response-operations-ui', log_level=app.config['LOGGING_LEVEL'])
 
+login_manager = LoginManager(app)
+login_manager.init_app(app)
+login_manager.login_view = "sign_in_bp.sign_in"
 
-from response_operations_ui import error_handlers  # NOQA # pylint: disable=wrong-import-position
-from response_operations_ui.views import collection_exercise, surveys  # NOQA # pylint: disable=wrong-import-position
-from response_operations_ui.views import info  # NOQA # pylint: disable=wrong-import-position
+
+@login_manager.user_loader
+def user_loader(user_id):
+    # TODO This will need to be replaced with a call to reddis to get the token
+    # as we can't leave that sort of information in the cookie.  Will be implemented
+    # down the line once uaa is sorted out.
+    return User(user_id)
+
+import response_operations_ui.views  # NOQA # pylint: disable=wrong-import-position
```

Comment:
why are these imports at the bottom?

Reply:
I **think** it's something to do with the way Flask is structured and the views need to imported after `app = Flask(__name__)`

Reply:
Yeah this is a common pattern in flask, you need to import the endpoints to register them but you need the app object to be created first. Otherwise you get weird cyclical import problems